extensions/amp-story/amp-story.md

```diff
@@ -254,53 +250,191 @@ The `bookend-config-src` value is a URL endpoint that returns data for the end s
   </amp-anim>
 </figure>
 
-#### Related articles
+#### Bookend Components
+
+The bookend is made up of a variety of components. These can be articles, call to action links, text, among others.
 
-The end screen displays related articles in sections. The heading for the section is obtained from the provided string key name (e.g., "More to Read"); the array of articles for that key are displayed for that section. The domain and favicon of each linked article are automatically parsed and fetched from the specified URL for each piece of content.
+They are configured in the `components` field of the configured JSON. See the [Example JSON response](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/amp-story.md#example-json-response) section below for an example.
 
-These are configured in the `related-articles` field of the response object.
+##### heading
+The `heading` component has a `text` field. It can be used to append a title to a group of articles.
+
+<amp-img alt="AMP story bookend heading component" layout="fixed"
```

Comment:
1) Fix your image file names - have incorrect spelling- "comopnent" vs "component".
2) Fix all the images as they are broken because incorrect URL and filename.

Should be like this:

```html
<amp-img alt="AMP story bookend heading component" layout="fixed"
src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-heading.png" width="386" height="123">
  <noscript>
    <img alt="AMP story bookend heading component" src="img/amp-story-bookend-component-heading.png" />
  </noscript>
</amp-img>
```